drivers/espi/espi_mchp_xec_host_v2.c

```diff
@@ -181,30 +191,75 @@ static void kbc0_ibf_isr(const struct device *dev)
 	struct espi_xec_data *const data =
 		(struct espi_xec_data *const)dev->data;
 
-
+#ifdef CONFIG_ESPI_PERIPHERAL_KBC_IBF_EVT_DATA
+	/* Chrome solution */
+	struct espi_event evt = { ESPI_BUS_PERIPHERAL_NOTIFICATION,
```

Comment:
would move ESPI_BUS_PERIPHERAL_NOTIFICATION to the next line so it can align with the others